README.md

```diff
@@ -107,6 +107,22 @@ curl localhost:8080/products/search \
       }'
 ```
 
+## Publish
+```shell
+curl localhost:8080/changes/publish \
```

Comment:
If we call this changes, it would look like change events. But this is not database change events. WDYT?

Reply:
Hmm yeah, not the best example choice. I was trying not to collide with other terms and overlooked this one. I'll figure out a different one.